lib/utils/isFirstNested.js

```diff
@@ -4,9 +4,44 @@
 module.exports = function(statement /*: postcss$node*/) /*: boolean*/ {
   const parentNode = statement.parent;
 
-  return (
-    parentNode !== undefined &&
-    parentNode.type !== "root" &&
-    statement === parentNode.first
-  );
+  if (parentNode === undefined || parentNode.type === "root") {
+    return false;
+  }
+
+  if (statement === parentNode.first) {
+    return true;
+  }
+
+  /*
+   * Search for the statement in the parent's nodes, ignoring comment
+   * nodes on the same line as the parent's opening brace.
+   */
+
+  const parentNodes = parentNode.nodes;
+  const firstNode = parentNodes[0];
+
+  if (firstNode.type !== "comment" || firstNode.raws.before.includes("\n")) {
+    return false;
+  }
+
+  const openingBraceLine = firstNode.source.start.line;
+
+  if (openingBraceLine !== firstNode.source.end.line) {
+    return false;
+  }
+
+  for (let i = 1; i < parentNodes.length; i++) {
+    const node = parentNodes[i];
+
+    if (node === statement) {
+      return true;
+    }
+
+    if (node.type !== "comment" || node.source.end.line !== openingBraceLine) {
+      return false;
+    }
+  }
+
+  /* istanbul ignore next: Should always return in the loop */
```

Comment:
Jest uses Istanbul for analyzing code coverage. That comment tells Istanbul not to count that line in the total code coverage since it's impossible to execute (the [documentation](https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md) is in the old Istanbul repo).